format/index.md

```diff
@@ -34,20 +34,21 @@ the header `Content-Type: application/vnd.api+json`.
 Clients **MUST** ignore all parameters for the `application/vnd.api+json`
 media type received in the `Content-Type` header of response documents.
 
```

Comment:
Yeah, obviously clients are not permitted to blow up in the face of unexpected parameters. I just thought that would be... well, obvious. Maybe it's indeed better to leave this intact just in case.

However, right now the paragraph is right between the two requirements to which "either of the requirements above" is intended to refer, so you may want to consider moving it to the very end or explicitly mentioning "request requirements above" or something like that.